docs/workflows/blocks_bundling.md

```diff
@@ -0,0 +1,118 @@
+# Bundling Workflows blocks
+
+To efficiently manage the Workflows ecosystem, a standardized method for building and distributing blocks is 
+essential. This allows users to create their own blocks and bundle them into Workflow plugins. A Workflow plugin 
+is essentially a Python library that implements a defined interface and can be structured in various ways. 
+
+This page outlines the mandatory interface requirements and suggests a structure for blocks that aligns with 
+the [Workflows versioning](/workflows/versioning) guidelines.
+
+## Proposed structure of plugin
+
+We propose the following structure of plugin:
+
+```
+.
+├── requirements.txt   # file with requirements
+├── setup.py           # use different package creation method if you like
+├── {plugin_name}
+│   ├── __init__.py    # main module that contains loaders
+│   ├── kinds.py       # optionally - definitions of custom kinds
+│   ├── {block_name}   # package for your block
+│   │   ├── v1.py      # version 1 of your block
+│   │   ├── ...        # ... next versions
+│   │   └── v5.py      # version 5 of your block
+│   └── {block_name}   # package for another block
+└── tests              # tests for blocks
+```
+
+## Required interface
+
+Plugin must only provide few extensions to `__init__.py` in main package
+compared to standard Python library:
+
+* `load_blocks()` function to provide list of blocks' classes (required)
+
+* `load_kinds()` function to return all custom kinds the plugin defines (optional)
```

Comment:
should we link to kinds page here / when we mention kinds?

Reply:
👍